src/BlazorUI/Bit.BlazorUI/Components/Slider/BitSlider.razor.cs

```diff
@@ -234,12 +234,12 @@ protected override void RegisterComponentClasses()
 
     protected override void OnInitialized()
     {
-        if (DefaultUpperValue.HasValue)
+        if (UpperValue.HasValue is false && DefaultUpperValue.HasValue)
```

Comment:
can't you use the `HasNoValue` extension method instead?

Reply:
No, UpperValue is double but the HasNoValue is an extension method for string type